### PR TITLE
Sample static results output

### PR DESCRIPTION
**Pull request by xdan-arch**
Testing static with this PR

### CODE REVIEW

**Comment by xdan-arch** [comment]
writing a comment from github

**Review by ghost — COMMENTED** [review]
:white_check_mark: A review job has been created and sent to the PullRequest network.  
*****  
Check the status or cancel PullRequest code review [here](https://daniel-b0af.pullrequest.tech/dash/projects/1/xDanielA/test-repo/reviews).

**Review by ghost — COMMENTED** [review]
this looks decent, we were testing multiple static outputs
[](#b7760bec-c914-4bcb-8bf0-14e82e298aa0 "PullRequest Meta [Do Not Modify]")
*****
Was this helpful? [Yes](https://daniel-b0af.pullrequest.tech/reviews/ratings/b7760bec-c914-4bcb-8bf0-14e82e298aa0/true) | [No](https://daniel-b0af.pullrequest.tech/reviews/ratings/b7760bec-c914-4bcb-8bf0-14e82e298aa0/false)

File: test2_copy2 copy.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+while(true){
+}
+
+
+console.log("A");
+debugger;
+console.log("B");
+debugger;
+console.log("C");
+debugger; console.log("D");debugger;if (foo == null) {bar();}
```
[the remark this answers is not in the document]
Comment (xdan-arch) [reply test2_copy2 copy.js:16]: Adding a comment from github

**Pull request merged**